<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowDecryption</class>
 <widget class="QWizard" name="WindowDecryption">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Расшифровать</string>
  </property>
  <widget class="QWizardPage" name="decrypt_file_or_message">
   <attribute name="pageId">
    <string notr="true">1</string>
   </attribute>
   <widget class="QCheckBox" name="checkBox_file">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Расшифровать файл</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_message">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Расшифровать сообщение</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_file">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Расшифровать файл</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_message">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Расшифровать сообщение</string>
    </property>
   </widget>
  </widget>
  <widget class="QWizardPage" name="insert_key">
   <attribute name="pageId">
    <string notr="true">3</string>
   </attribute>
   <widget class="QTextEdit" name="text_isert_key">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>158</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="button_ok_insert_key">
       <property name="text">
        <string>Принять</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_cancel_insert_key">
       <property name="text">
        <string>Сбросить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_key">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>391</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Drop" name="name_file_key">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="button_insert_file">
       <property name="text">
        <string>Вставить ключ из файла</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_insert_key">
       <property name="text">
        <string>Ввести ключ в поле</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWizardPage" name="show_name_file_or_message">
   <attribute name="pageId">
    <string notr="true">4</string>
   </attribute>
   <widget class="QTextEdit" name="textEdit_your_file_or_message_for_decrypt">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_your_file_or_message_for_decrypt">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ваше сообщение или файл который Вы хотите расшифровать</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_key_for_decrypt">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>361</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_key_for_decrypt">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_decrypt">
    <property name="geometry">
     <rect>
      <x>264</x>
      <y>160</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Расшифровать</string>
    </property>
   </widget>
  </widget>
  <widget class="QWizardPage" name="insert_name_file_or_message">
   <attribute name="pageId">
    <string notr="true">2</string>
   </attribute>
   <widget class="QLabel" name="label_insert_name_fale_or_message">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Встаить файл или сообщение</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_message">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>331</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="Drop" name="area_name_file">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWizardPage" name="result">
   <attribute name="pageId">
    <string notr="true">5</string>
   </attribute>
   <widget class="QTextEdit" name="textEdit_result">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>351</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_your_decrypt_file_or_message">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>351</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ваш расшифрованный файл или сообщение</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Drop</class>
   <extends>QLineEdit</extends>
   <header>drop.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
